3rdparty/obiwarp/obiwarp.h

```diff
@@ -0,0 +1,46 @@
+#ifndef OBIWARP_H
+#define OBIWARP_H
+
+#include <vector>
+#include <iostream>
+#include <assert.h>
+
+#include "vec.h"
+#include "mat.h"
+#include "dynprog.h"
+
+using namespace std;
+
+class ObiWarp{
+public:
+    ObiWarp(string score="cor",bool local = false, float factor_diag = 2.f, float factor_gap = 1.f, 
+            float gap_init = 0.2f,float gap_extend = 3.4f, float init_penalty = 0.f, 
+            float response = 100.f, bool nostdnrm = false  );
```

Comment:
Should we not use astruct here ?
You have added a struct ,obiParams, in mzAligner.cpp , can we not pass that struct here ?

Reply:
@rish9511 done!